src/schema/server_settings.py

```diff
@@ -106,7 +105,7 @@ class ServerSettings(BaseModel):
     default_function_capable_llm_max_tokens: int = Field(512, description="")
 
     # Language Generation Settings - Story telling
-    default_story_model: str = Field("gpt-3.5-turbo", description="")
+    default_story_model: str = Field("gpt-4-1106-preview", description="")
```

Comment:
question: do we believe the 1106-preview model is ready for primetime (rate limits, mainly)?

Reply:
I guess it isn't.

<img width="575" alt="image" src="https://github.com/steamship-core/ai-adventure-agent/assets/4643013/50905e26-4fd1-425f-8891-c221935b547f">